eraser-map.yaml

```diff
@@ -861,7 +861,6 @@ styles:
     icons:
         base: points
         texture: pois
-        interactive: true
```

Comment:
@bcamper Not for this PR, but can this be made to work? It's confusing why this doesn't inherit.